src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -128,6 +131,24 @@ public int
                 );
         }
 
+        [Fact]
+        [WorkItem(217398, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=217398")]
+        public void LexerTooManyBadTokens_Bug217398()
```

Comment:
Bug number is not needed in the method name since it's included in the `[WorkItem]`.